pkg/cluster/internal/create/create.go

```diff
@@ -159,6 +160,10 @@ func Cluster(logger log.Logger, p providers.Provider, opts *ClusterOptions) erro
 		return err
 	}
 
+	if err = crafting.PatchKubeProxy(p, opts.Config.Name); err != nil {
```

Comment:
Can we move this into crafting provider's `Provision` impl?

Reply:
Probably needs a bit more work to dig how the kind container image works and try to wrap the entrypoint from there.

Reply:
kube-proxy manifest is hard-coded in kubeadm: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/addons/proxy/manifests.go#L82

Reply:
Not sure if `--patches` flag or `--skip-phases addon/kube-proxy` can be used for `kubeadm init` command

Reply:
Probably the best way to go is run `kubectl patch` right after `kubeadm init`.

Reply:
https://github.com/pigfall/kind/blob/27aadd72d0f6776bce9a2151fc95841493a5f52a/pkg/cluster/internal/create/create.go#L119 .  I add a patch action after the `kubeadm init` action